d2l-button-behavior.html

```diff
@@ -102,14 +102,27 @@
 		}
 
 	},
+
 	ready: function() {
-		this.addEventListener('tap', function(e) {
-			var button = e.currentTarget;
-			if (button && button.disabled) {
-				e.stopPropagation();
-			}
-		}, true);
+		this._handleClick = this._handleClick.bind(this);
+	},
+
+	attached: function() {
+		Polymer.RenderStatus.afterNextRender(this, function() {
+			this.addEventListener('click', this._handleClick, true);
+		}.bind(this));
```

Comment:
Was the issue that previously it was using `tap` and not `click`, or the `e.currentTarget` stuff?

Reply:
With P3, the `tap` was not working as in P1.  In my tests, the tap handler wasn't being invoked at all.  I double checked the new logic on iPhone and of course all the desktop browsers as well.